charts/argo-cd/README.md

```diff
@@ -105,6 +105,11 @@ For full list of changes please check ArtifactHub [changelog].
 
 Highlighted versions provide information about additional steps that should be performed by user when upgrading to newer version.
 
+### 5.32.0
+This version changed supported Kubernetes version from `>=1.22.0-0` to `>=1.24.0-0`.
+
+Please see more information about EoL: [Kubernetes EoL][k8s EoL].
+
```

Comment:
Though Argo CD has specific K8s version support, I wonder this would give a lot of users effects.
so I just updated as following K8s EoL.
https://argo-cd.readthedocs.io/en/stable/operator-manual/installation/#supported-versions

<img width="806" alt="image" src="https://user-images.githubusercontent.com/30188755/235917276-cfd1334a-3753-4e8d-bfc3-554158998bdb.png">